core/diag.c

```diff
@@ -298,6 +298,9 @@ void diag(int call_code) {
           get_time_per_step(TIMER_ALL));
 #if ELECTRONS
       fprintf(ener_file, "%15.8g ", get_time_per_step(TIMER_ELECTRON));
+#endif
+#if NEUTRINO_OSCILLATIONS || LOCAL_ANGULAR_DISTRIBUTIONS
```

Comment:
What is the difference between these two conditions?

Reply:
One can compute G, A, B, etc without actually running the oscillations code. So `NEUTRINO_OSCILLATIONS` requires `LOCAL_ANGULAR_DISTRIBUTIONS` but not vice versa. Would be sufficient to include only the latter, but I was being careful.